File: help/eng_gb/exercicios da ficha_11/Ficha11_008/Ficha11_008.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha11_008 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("height : " );
        int Height = Integer.valueOf(keyb.nextLine());
        for( int i = 1; i < Height + 1; i = i + 1 ) {
            line(i ) ;
        }
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void line( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            System.out.print("*" );
        }
        System.out.print("\n" );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:22

File: help/eng_gb/exercicios da ficha_11/Ficha11_008/Ficha11_008.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "height : " ;
    $Height = intval(isset($_POST["Height"])? $_POST["Height"] : isset($_GET["Height"]) ? $_GET["Height"] : 0);
    
    echo $Height . "<br/>";// print input 
    for( $i = 1; $i < $Height + 1; $i = $i + 1 ) {
        line($i ) ;
    }
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function line($n) {
    for( $i = 0; $i < $n; $i = $i + 1 ) {
        echo  "*" ;
    }
    echo  "\n" ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 20:22

?>